### PR TITLE
WT-7542 Add a Python test to reconfigure zstd compression level after restart

### PR DESCRIPTION
**Pull request by mm-ng**

### CODE REVIEW

File: test/suite/test_compress02.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        count = 0
+        for k, v in cursor:
+            self.assertEqual(v, check_value)
+            count += 1
+        session.commit_transaction()
+        self.assertEqual(count, nrows)
+
+    def test_compress02(self):
+
+        ds = SimpleDataSet(self, self.uri, 0, key_format="S", value_format="S",config='log=(enabled=false)')
```
Comment (bvpvamsikrishna) [inline test/suite/test_compress02.py:78]: I think we need to pass "block_compress=zstd" in the config. Then only it will use zstd as a compressor.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Review by bvpvamsikrishna — APPROVED** [review]
lgtm

**Comment by mm-ng** [comment]
evergreen retry

**Comment by mm-ng** [comment]
evergreen retry

**Pull request merged**